qiskit/circuit/instruction.py

```diff
@@ -160,6 +160,7 @@ def to_mutable(self):
         return self.copy()
 
     @property
+    @deprecate_func(since="1.3.0", removal_timeline="in 2.0.0", is_property=True, stacklevel=2)
```

Comment:
I believe that 2 is the default stack level, right? It doesn't hurt but it made me think for a second, so maybe:
```suggestion
    @deprecate_func(since="1.3.0", removal_timeline="in 2.0.0", is_property=True)
```

Reply:
Oh good call, this was leftover from me playing around with the deprecation warning in different settings. We end up accessing this in a few different ways internally and I wanted to see what was user visible or not depending on circuit usage. I tried changing this value for different scenarios and settled on 2 being the best choice and left it explicit like this. I'll remove the kwarg so people realize it's the default.